src/ActionMenu2.tsx

```diff
@@ -70,7 +70,7 @@ export type MenuButtonProps = ButtonProps
 const MenuButton = React.forwardRef<AnchoredOverlayProps['anchorRef'], ButtonProps>((props, anchorRef) => {
   return (
     <Anchor ref={anchorRef}>
-      <Button {...props} />
+      <Button type="button" {...props} />
```

Comment:
Good call. This was bound to break somebody's form.

Reply:
Out of curiosity, shouldn't this be applied by default to underlying property if the underlying tag is `button`?

Reply:
@rezrah You mean we should move this to `Button` component instead? (default for html button is type=submit)